inc/formanswer.class.php

```diff
@@ -75,7 +75,7 @@ class PluginFormcreatorFormAnswer extends CommonDBTM
 
    public static function getStatuses() {
       return [
-         self::STATUS_WAITING  => __('Waiting', 'formcreator'),
+         self::STATUS_WAITING  => __('Waiting for approval'),
```

Comment:
In this case, the variable name is ambiguous

Reply:
Fixed in [d53bd46](https://github.com/pluginsGLPI/formcreator/pull/3455/commits/d53bd4645b2ce3e43c66501a56aab5200bd333ce)